Makefile

```diff
@@ -111,15 +111,13 @@ help: ## Display this help.
 # Directories.
 
 ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
-BIN_DIR := bin
-TOOLS_DIR := hack/tools
-TOOLS_BIN_DIR := $(abspath $(TOOLS_DIR)/$(BIN_DIR))
+BIN_DIR := $(abspath bin)
 GO_INSTALL := ./scripts/go_install.sh
 CONVERSION_GEN_VER := v0.25.0
 CONVERSION_GEN_BIN := conversion-gen
 # We are intentionally using the binary without version suffix, to avoid the version
 # in generated files.
-CONVERSION_GEN := $(abspath $(TOOLS_BIN_DIR)/$(CONVERSION_GEN_BIN))
+CONVERSION_GEN := $(abspath $(BIN_DIR)/$(CONVERSION_GEN_BIN))
```

Comment:
BIN\_DIR is already made absolute on line 114 so no need to do it again here.  
`CONVERSION_GEN := $(BIN_DIR)/$(CONVERSION_GEN_BIN)`

Reply:
done